libs/blocks/icon-block/icon-block.css

```diff
@@ -70,7 +100,7 @@
   margin-bottom: 0;
 }
 
```

Comment:
Since you are using the decorateButton() we probably want to add margin for `supplemental-text` incase it's present. This would be a paragraph after the action area...

```
.icon-block .foreground .text .supplemental-text {
  margin-top: var(--spacing-s);
}
```
Open to ideas here, we could also isolate that need for that in the specific block(s). [marquee]

Reply:
sweet added!